runtime/lang/menu_fi_fi.latin1.vim

```diff
@@ -1,6 +1,6 @@
 " Menu Translations:	Finnish
 " Maintainer:		Flammie Pirinen <flammie@iki.fi>
-" Last Change:		2020 Apr 23
+" Last Change:		1 May 2024
```

Comment:
Please keep the existing style of the Last Change header, so change it to `2024 May 02`